docs/edit-profile.md

```diff
@@ -29,5 +29,5 @@ You need the following to edit your TechPass profile via the TechPass portal:
 
 4. Make the required changes and click **Save**.
 
-?> <br>- You can't edit the **Agency** name to which your TechPass is linked.<br>- To edit any typo in your email address, create a [service request](https://go.gov.sg/seed-techpass-support) with us.
+?> <br>- You can't edit the **Agency** name to linked to your TechPass.<br>- To edit any typo in your email address, create a [service request](https://go.gov.sg/seed-techpass-support) with us.
```

Comment:
You can't edit the **Agency** name linked to your TechPass.

would removing _to_ sound bettter?

should we even mention typo? 

_To edit your email address, create a [service request](https://go.gov.sg/seed-techpass-support) with us._

Reply:
the english is wrong.

Reply:
@kpslp , the main focus here is to tell the user they can't directly edit the email address for typos and must raise an SR. Apparently, the SR is only for typo issues. They can't request to change it to a new email address.  

However, i can include that they can't delete the email address.

Reply:
English fixed 

"You can't edit or delete the **Agency** name linked to your TechPass.<br>- To edit any typos in your email address, create a [service request](https://go.gov.sg/seed-techpass-support) with us."